syntax = "proto3";
package mars.stn;


option java_package = "win.liyufan.im.proto";
option java_outer_classname = "CreateGroupRequestOuterClass";

//option objc_class_prefix = "LYF";

import "group.proto";
import "message_content.proto";

message CreateGroupRequest {
  Group group = 1;
  repeated int32 to_line = 2;
  MessageContent notify_content = 3;
}
